openmeter/ent/schema/billing.go

```diff
@@ -597,6 +602,15 @@ func (BillingInvoice) Fields() []ent.Field {
 				dialect.Postgres: "char(26)",
 			}),
 
+		// App IDs
```

Comment:
don't we need a `tax_app_external_id` too?

Reply:
Eventually we will but tax is a heavier lift, as we need to implement calculations, tax db tables and recalculation avoidance, so that's better kept seperate